composer.json

```diff
@@ -1,5 +1,16 @@
 {
+    "name": "lndrest/composer",
```

Comment:
how are other composer packages named? what is the naming convention of those packages? (I don't think a composer package should be called something "composer")